### ISSUE TITLE
consider a non-red color for when we expand for more info

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#392]
When someone clicks on the blue (+) button next to a domain on the HTTPS or (upcoming) a11y sections, the blue (+) turns into a red (-) button.  I understand it's not intended this way, but that seems to denote negativity (e.g. `Fail` or `Error`).  It does this regardless of the data.  

I'd be game with pretty much any other color.  Thoughts?  

![screen shot 2016-03-16 at 5 27 13 pm](https://cloud.githubusercontent.com/assets/633088/13829250/61d243dc-eb9c-11e5-9e75-8fd763787901.png)

**Comment by anand-bhat** [comment]
Reported earlier at https://github.com/18F/pulse/issues/253

**Comment by konklone** [comment]
Mooted by #518.
